src/main/java/org/sciborgs1155/robot/Hopper/HopperIO.java

```diff
@@ -0,0 +1,15 @@
+package org.sciborgs1155.robot.Hopper;
+
+import monologue.Logged;
+
+public interface HopperIO extends Logged {
+
+  public double getSpeed();
```

Comment:
so after the changes in the other comments, i think this should have getSpeed() or speed(), and a setVoltage(double voltage), and that's it (i might be forgetting something)